### PR TITLE
Added changed method: change activitystatus, added QuickEdit calendar…

### PR DESCRIPTION
**Pull request by adudekyeti**
Added changed method: change activitystatus, added QuickEdit calendar event

### CODE REVIEW

File: modules/Calendar/views/EventForm.php [hunk 1]
```diff
@@ -89,4 +89,18 @@ public function postProcessAjax(\App\Request $request)
...
 		$viewer->view('Extended/EventForm.tpl', $request->getModule());
 	}
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public function getFooterScripts(\App\Request $request)
+	{
+		$jsFiles = parent::getFooterScripts($request);
+		if (AppConfig::module('Calendar', 'CALENDAR_VIEW') === 'Extended') {
```
Comment (rskrzypczak) [inline modules/Calendar/views/EventForm.php]: this view is available only in Extended calendar

File: modules/Calendar/views/QuickEditAjax.php [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+/**
+ * QuickEdit view for module Calendar.
+ *
+ * @package   Action
+ *
+ * @copyright YetiForce Sp. z o.o.
+ * @license   YetiForce Public License 3.0 (licenses/LicenseEN.txt or yetiforce.com)
+ * @author    Arkadiusz Dudek <a.dudek@yetiforce.com>
+ */
+class Calendar_QuickEditAjax_View extends Vtiger_QuickCreateAjax_View
```
Comment (rskrzypczak) [inline modules/Calendar/views/QuickEditAjax.php]: ?

File: public_html/layouts/basic/modules/Vtiger/resources/Header.js [hunk 3]
```diff
@@ -482,6 +482,37 @@ $.Class("Vtiger_Header_Js", {
 			});
 		});
 	},
+	/**
+	 * Show quick edit calendar entry
+	 * @param moduleName
+	 * @param params
+	 */
+	quickEditModule(moduleName, params) {
```
Comment (rskrzypczak) [inline public_html/layouts/basic/modules/Vtiger/resources/Header.js]: ?

**Comment by adudekyeti** [comment]
To many conflicts, created new pull request: #8259

**Pull request closed without merging**